website/docs/d/alert_configuration.html.markdown

```diff
@@ -226,13 +230,13 @@ Notifications to send when an alert condition is detected.
     - `WEBHOOK`
     - `MICROSOFT_TEAMS`
 
+* `notifier_id` - The notifier id is a system-generated unique identifier assigned to each notification method. This is needed when updating third-party notifications without requiring explicit authentication credentials.
```

Comment:
```suggestion
* `notifier_id` - The notifier id is a system-generated unique identifier assigned to each notification method. This is needed to update third-party notifications without explicit authentication credentials.
```

Reply:
given that this was copied directly from our API docs (https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration within notifications), would it be better to leave as is for consistency?